graf2d/gpad/src/TPad.cxx

```diff
@@ -1225,13 +1225,12 @@ Int_t TPad::DistancetoPrimitive(Int_t px, Int_t py)
 /// Automatic pad generation by division.
 ///
 ///  - The current canvas is divided in nx by ny equal divisions (pads).
-///  - xmargin defines the horizontal spacing around each pad as a percentage of the canvas
-///    width. Therefore, the distance between two adjacent pads along the x-axis is equal
-///    to twice the xmargin value.
-///  - ymargin defines the vertical spacing around each pad as a percentage of the canvas
-///    height. Therefore, the distance between two adjacent pads along the y-axis is equal
-///    to twice the ymargin value.
+///  - xmargin defines the horizontal spacing between each pad as a percentage of the canvas
+///    width.
+///  - ymargin defines the vertical spacing between each pad as a percentage of the canvas
+///    height.
 ///  - color is the color of the new pads. If 0, color is the canvas color.
+///  - All pads are contained within the inner area defined by the canvas margins.
 ///
```

Comment:
```suggestion
/// - Note that, if you don't have a background color of your pad, the spacing between pads might look larger than specified,
/// since in the default case, each pad has internally an empty space on the right equal to the space filled on the left for the y axis labels 
```

Maybe sth along these lines would be helpful, since most people don't put background color on their pads.